integrations/atlassian/jira/event.go

```diff
@@ -106,10 +107,17 @@ func (h handler) handleEvent(w http.ResponseWriter, r *http.Request) {
 		return int(f)
 	})
 
+	u, err := transformIssueURL(jiraEvent, l)
+	if err != nil {
+		l.Error("Failed to transform issue URL", zap.Error(err))
```

Comment:
1. Not all Jira events relate to issues
2. It's easier to understand the initial meaning/context of this log message in Datadog when you specify it's related to Jira
```suggestion
		l.Error("Failed to transform Jira URL", zap.Error(err))
```